cli/import-jdl.js

```diff
@@ -229,14 +229,15 @@ const generateEntityFiles = (generator, entity, inFolder, env, shouldTriggerInst
         });
     } else {
         /* Traditional entity only generation */
+        const doneEntity = doneFactory(`JHipster entity ${entity.name} generation completed!`);
```

Comment:
I think we already print out the same message at; 

https://github.com/jhipster/generator-jhipster/blob/b9240e6efb988b632e7115164051b7d380effefc/generators/entity-client/index.js#L64-L73

Reply:
Yep. But we should print the jdl process has finished.
I will change the message.

A success message for jdl will not be easy...

Reply:
I was wondering why all messages was grouped.
All entities are called in the same yeoman-environment =/